cpp/log/cert.cc

```diff
@@ -264,10 +293,10 @@ Cert::Status Cert::HasCriticalExtension(int extension_nid) const {
 }
 
 
-Cert::Status Cert::HasBasicConstraintCATrue() const {
+StatusOr<bool> Cert::HasBasicConstraintCATrue() const {
   if (!IsLoaded()) {
     LOG(ERROR) << "Cert not loaded";
-    return ERROR;
+    return ERROR_STATUS;
```

Comment:
Returning `Status::UNKNOWN`? That's a bit unfriendly, I try to avoid it, as it has no message string... You can also get the benefit of `util::Status` better, letting you have the message be something slightly more specific for human beings, like "certificate not loaded", or "NID not found".

In `CertStatusToStatusOrBool`, you won't have much of a chance, though, but since it might be the only use of `ERROR_STATUS`, you could just use `util::Status::UNKNOWN` directly there?